overlord/snapstate/snapstate.go

```diff
@@ -52,6 +52,10 @@ const (
 	// for use in the interim time while we have the backward compatible
 	// support
 	firstInterimUsableFlagValue
+
+	// RollbackMode means the given snap got rolled back
+	RollbackMode
+
```

Comment:
not sure we should call Mode something like this also because it seems it's SnapSetup only flag, so it should follow my suggestion in the comment below in the code (if it's really a mode like trymode then the previous definition should go away and we should grab the value of firstInterimUsableFlagValue here instead of the one following)